Sources/ShapedArray/debug.swift

```diff
@@ -1,5 +1,5 @@
 public func debugLog(_ str: String) {
-    #if DEBUG
+#if DEBUG
     print(str)
-    #endif
+#endif
 }
```

Comment:
Let's remove the `debugLog` entirely. The default behavior of TensorFlow's `_RuntimeConfig.printsDebugLog` was to disable it by default. We don't want messages printed to the console unless the developer explicitly calls `print(...)`, even in debug mode. Two other locations call into `debugLog`, and the should be deleted as well.

https://github.com/s4tf/s4tf/blob/a31f79a7a8e92ff708b1034286a23510b23af5f3/Sources/TensorFlow/Core/Runtime.swift#L81-L85

Reply:
To remove this unrelated change from the conversation, I can extract it into #4. @JaapWijnen you do not have to incorporate my suggestion into your branch.